app/main.ipynb

```diff
@@ -32,15 +37,46 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
+   "execution_count": 79,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "248"
+     },
+     "execution_count": 79,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your code here"
+    "conn = sqlite3.connect(\"../db.sqlite3\")\n",
+    "\n",
+    "query_order = \"SELECT * FROM restaurant_order;\"\n",
+    "df_order = pd.read_sql_query(query_order, conn)\n",
+    "\n",
+    "query_orderitem = \"SELECT * FROM restaurant_orderitem;\"\n",
+    "df_orderitem = pd.read_sql_query(query_orderitem, conn)\n",
+    "\n",
+    "query_product = \"SELECT * FROM restaurant_product;\"\n",
+    "df_product = pd.read_sql_query(query_product, conn)\n",
+    "\n",
+    "combined_df = pd.concat([df_order, df_orderitem, df_product], ignore_index=True)\n",
+    "\n",
+    "combined_df.to_csv(\"combined_data.csv\", index=False)\n",
+    "\n",
+    "conn.close()\n",
+    "\n",
+    "df = pd.read_csv(\"combined_data.csv\")\n",
+    "len(df)"
```

Comment:
Incorrect result and implementation. In the task description it's mentioned that there should be 74818 rows + in your case all 3 tables and independent and you just merge them, but they have smth in common that can be used here. And because of it all other tasks have incorrect results :(